protocol/0028-GOVE-governance.md

```diff
@@ -169,16 +169,13 @@ We introduce 2 new commands which require consensus (needs to go through the cha
 ## Types of proposals
 
 Every proposal transaction contains the following common fields:
-- a link to a text file in markdown format
-- a cryptographically secure hash (SHA3-512) of the text so that viewers can check that the text hasn't been changed since the proposal was submitted and
-- a description field to show a short title / something in case the link goes offline.
-
-The protocol (Vega core) is not expected to verify that the hash corresponds to the contents of the linked file. It is expected that any client tool that allows voting will do this at client level.
+- a title field to briefly describe the proposal that can be used when listing proposals
+- a description field to contain the additional details behind the proposal as well as some rationale
+If more details is required about the proposal, a proposer can reference immutable external resources in the proposal description. e.g. [IPFS](https://en.wikipedia.org/wiki/InterPlanetary_File_System) links.
```

Comment:
Should we verify the content of the description for link pointing *mutable* resources ?

Reply:
we verify nothing in the contents.